android/src/main/java/net/glxn/qrgen/android/QRCode.java

```diff
@@ -1,180 +1,193 @@
-     *
-     * @param width  the width in pixels
-     * @param height the height in pixels
-     * @return the current QRCode object
-     */
-    public QRCode withSize(int width, int height) {
-        this.width = width;
-        this.height = height;
-        return this;
-    }
-
-    /**
-     * Overrides the default charset by supplying a {@link com.google.zxing.EncodeHintType#CHARACTER_SET} hint to {@link
-     * com.google.zxing.qrcode.QRCodeWriter#encode}
-     *
-     * @return the current QRCode object
-     */
-    public QRCode withCharset(String charset) {
-        return withHint(EncodeHintType.CHARACTER_SET, charset);
-    }
-
-    /**
-     * Overrides the default error correction by supplying a {@link com.google.zxing.EncodeHintType#ERROR_CORRECTION} hint to
-     * {@link com.google.zxing.qrcode.QRCodeWriter#encode}
-     *
-     * @return the current QRCode object
-     */
-    public QRCode withErrorCorrection(ErrorCorrectionLevel level) {
-        return withHint(EncodeHintType.ERROR_CORRECTION, level);
-    }
-
-    /**
-     * Sets hint to {@link com.google.zxing.qrcode.QRCodeWriter#encode}
-     *
-     * @return the current QRCode object
-     */
-    public QRCode withHint(EncodeHintType hintType, Object value) {
-        hints.put(hintType, value);
-        return this;
-    }
-
-    /**
-     * Returns a {@link android.graphics.Bitmap} without creating a {@link java.io.File} first.
-     *
-     * @return {@link android.graphics.Bitmap} of this QRCode
-     */
-    public Bitmap bitmap() {
-        try {
-            return MatrixToImageWriter.toBitmap(createMatrix(text), matrixToImageConfig);
-        } catch (WriterException e) {
-            throw new QRGenerationException("Failed to create QR image from text due to underlying exception", e);
-        }
-    }
-
-    @Override
-    public File file() {
-        File file;
-        try {
-            file = createTempFile();
-            MatrixToImageWriter.writeToFile(createMatrix(text), imageType.toString(), file, matrixToImageConfig);
-        } catch (Exception e) {
-            throw new QRGenerationException("Failed to create QR image from text due to underlying exception", e);
-        }
-
-        return file;
-    }
-
-    @Override
-    public File file(String name) {
-        File file;
-        try {
-            file = createTempFile(name);
-            MatrixToImageWriter.writeToFile(createMatrix(text), imageType.toString(), file, matrixToImageConfig);
-        } catch (Exception e) {
-            throw new QRGenerationException("Failed to create QR image from text due to underlying exception", e);
-        }
-
-        return file;
-    }
-
-    @Override
-    protected void writeToStream(OutputStream stream) throws IOException, WriterException {
-        MatrixToImageWriter.writeToStream(createMatrix(text), imageType.toString(), stream, matrixToImageConfig);
-    }
+	protected final String text;
```

Comment:
There is still quite a lot more changes than are relevant. Can you check your whitespace settings. erhaps you are mixing tabs and spaces?

Reply:
Looks like you have 8 indentions instead of 4. If you adjust this I think we'll see a much smaller diff.